boards/ssrc/saluki/init/rc.board_defaults

```diff
@@ -7,3 +7,8 @@ set OUTPUT_MODE pwm_esc
 set AUX_MODE none
 
 pwm_esc start
+
+param set-default COM_PREARM_MODE 1
```

Comment:
I think this is default, no need to set it

Reply:
> I think this is default, no need to set it

Yes, it's default.
https://docs.px4.io/master/en/advanced_config/prearm_arm_disarm.html#arming-sequence-pre-arm-mode-safety-button

Reply:
> > I think this is default, no need to set it
> 
> Yes, it's default. https://docs.px4.io/master/en/advanced_config/prearm_arm_disarm.html#arming-sequence-pre-arm-mode-safety-button

Actually this documentation is outdated. COM_PREARM_MODE is 0 (== disabled) by default.
https://github.com/PX4/PX4-Autopilot/commit/c6a016104713846a72aa2bcaf18dcda6bd54367c#diff-98b4e0f87126f3486c8dda8f693ee45c682a856f17958aed5a33d51e5fd27766